pkgs/applications/networking/instant-messengers/franz/generic.nix

```diff
@@ -24,19 +24,25 @@
 , xdg-utils
 , mesa
 , libappindicator-gtk3
+, libdrm
+, libGL
+, libxkbcommon
+, wayland
+, pipewire
 }:
 
 # Helper function for building a derivation for Franz and forks.
 
-{ pname, name, version, src, meta, extraBuildInputs ? [] }:
+{ pname, name, version, src, meta, extraBuildInputs ? [], commandLineArgs ? "" }:
 
 stdenv.mkDerivation rec {
   inherit pname version src meta;
 
   # Don't remove runtime deps.
   dontPatchELF = true;
+  dontStrip = true;
 
-  nativeBuildInputs = [ autoPatchelfHook makeWrapper wrapGAppsHook dpkg ];
+  nativeBuildInputs = [ patchelf (wrapGAppsHook.override { inherit makeWrapper; }) dpkg ];
```

Comment:
Why are we overwriting wrapGappsHook?

Reply:
This is needed because the binary wrapper doesn't support variable expansion which is used below.